docs/xbcloud-binary-fifo-datasink.md

```diff
@@ -0,0 +1,62 @@
+# FIFO datasink
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+[Percona XtraBackup 8.0.33-28](release-notes/8.0/8.0.33-28.0.md) implements a new datasink that uses `FIFO` (named pipes). With FIFO datasink feature, you can benefit from faster backups by streaming data in parallel from XtraBackup to `xbcloud`.
+
+## Stream to object storage
+
+When taking a backup, you can save the files locally or stream the files to either different servers or object storage. 
+
+XtraBackup streams data to object storage using [`xbcloud`](xbcloud-binary-overview.md) as follows:
+
+* XtraBackup spawns multiple copy threads. 
+
+* Each copy thread reads a data chunk from a specific file.
+
+* Each copy thread writes the data chunk to a pipe (STDOUT).
+
+* A read thread reads each chunk of STDIN (standard input device) data. 
+
+* Xbcloud uploads the chunks to an object storage using an async request, and a callback is added to an event handler list.
+
+* The xbcloud event handler executes the callback depending on the response from the object storage (success or failure). 
+
+![image](_static/backup-streamed-to-object-storage.png)
+
+XtraBackup streaming has a limitation of 1.8 Gbps. XtraBackup writes one thread to STDOUT simultaneously using a mutex. This operation ensures that a chunk is written from the beginning to the end. Uploading chunks on a Wide Area Network (WAN) experiences no lag, but streaming backups using a Local Area Network (LAN) can experience a lag because those configurations are designed for at least 10Gbps.
+
+XtraBackup streams data to `xbcloud` using FIFO as follows:
+
+* Multiple FIFO files are created, each responsible for storing the data from XtraBackup to `xbcloud`.
+
+* Each xtrabackup copy thread is bound to a FIFO file in a round-robin fashion.
+
+* Xbcloud has multiple pair of read threads and event handlers.
+
+![image](_static/fifo-datasink.png)
+
+## Use FIFO datasink
+
+[Percona XtraBackup 8.0.33-28](release-notes/8.0/8.0.33-28.0.md) implements the following options:
+
+* `--fifo-streams=#` - specifies the number of FIFO files to use for parallel data stream. To disable FIFO datasink and send stream to STDOUT, set `--fifo-streams=1`. The default value is `1` (STDOUT) to ensure backward compatibility.
+* `--fifo-dir=name` - specifies a directory to write Named Pipe.
+* `--fifo-timeout=#` - specifies the number of seconds to wait for the other end to open the stream for reading. The default value is `10` seconds.
+
+When using `--parallel-stream=1` (default and current behaviour), we have a single line command with xtrabackup writing to STDOUT and xbcloud reading from STDIN.
+
+To run FIFO datasink, we should either invoke both commands (`--fifo-streams=#` and `--parallel-stream=1`) in separate terminal sessions or run xtrabackup in the background.
+
+### Example of FIFO datasink usage 
```

Comment:
example missing? and the below text should be under a heading like 'Performance improvement'?

Reply:
updated